stubs/PDO.stub

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class PDO
+{
+    /**
+     * @template RowType of array
+     *
+     * @param non-empty-string $query
+     * @param class-string|object|int $classOrColNo
```

Comment:
[this method has a few different variants](https://www.php.net/manual/en/pdo.query.php) therefore I created a union of all possible types.